hipcheck/src/config.rs

```diff
@@ -806,7 +833,7 @@ impl WeightTreeNode {
 	// @Temporary - until policy file impl'd and integrated, we hard-code
 	// the policy for our analyses
 	pub fn with_hardcoded_expr(&self) -> AnalysisTreeNode {
-		let expr = Expr::Primitive(Primitive::Bool(false));
+		let expr = "true".to_owned();
```

Comment:
Note that `true` is not valid Policy Expression syntax; `#t`/`#f` is the syntax for a literal boolean.
I'm not sure how temporary this code is expected to be; maybe this will never be evaluated.

Reply:
I've submitted a PR (#338) to fix this.